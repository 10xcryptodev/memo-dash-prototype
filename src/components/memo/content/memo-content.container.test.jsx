import React from 'react'
import configureStore from 'redux-mock-store'
import { shallow } from 'enzyme'
import MemoContentContainer from './memo-content.container'

import testUsers from '../../../test-utils/test-users'

describe('<MemoContentContainer />', () => {
  let store
  let testUser
  beforeEach(() => {
    testUser = testUsers['alice']
    // Mock store
    const mockStore = configureStore()
    store = mockStore({
      user: { currentUser: testUser.username }
    })

    const div = document.createElement('div')
    document.body.appendChild(div)
  })

  describe('Shallow rendering', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<MemoContentContainer />, { context: { store } })
      expect(wrapper).toMatchSnapshot()
    })
  })
})
