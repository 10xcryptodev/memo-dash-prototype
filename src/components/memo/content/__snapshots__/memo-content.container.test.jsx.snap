// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MemoContentContainer /> Shallow rendering renders without crashing 1`] = `
<MemoComponent
  onEditClicked={[Function]}
  onGoToProfileClicked={[Function]}
/>
`;
